### ISSUE TITLE
[TRACKING ISSUE] Color contrast problems in the UI

### ISSUE THREAD

**Issue by jglovier** [HospitalRun/hospitalrun-frontend#972]
**Expected behavior:**
All text in the UI should pass at least [WCAG AA](https://www.w3.org/WAI/WCAG20/quickref/) contrast accessibility. A simple test of text-color on background color on [this color accessibility calculator](http://webaim.org/resources/contrastchecker/) will tell whether any existing pairing of font on background passes. Many of our instances do, and a few do not.

**Actual behavior:**
Currently, there are a number of areas that need to improve to meet AA standard.
Including:
- [ ] _[add checklist of instances to address here]_

**Screenshots (if applicable):**

Some offenders:

---

![image](https://cloud.githubusercontent.com/assets/1319791/23594767/9867d69a-01ea-11e7-9216-441bfc1004f1.png)

---

![image](https://cloud.githubusercontent.com/assets/1319791/23594873/816ec22c-01eb-11e7-9f45-6f4ab2781cc6.png)

---

![image](https://cloud.githubusercontent.com/assets/1319791/23594904/bbafa8de-01eb-11e7-8731-55d18bdeacbb.png)

---

**How to solve with tooling:**

Some ways we can address this going forward are with tooling.
- [ ] Establish a pattern library and styleguide that removes the possibility of introducing any new failing color styles
- [ ] Add a test for contrast (will need to research options more)

**Comment by jglovier** [comment]
At this point what I could really use is some help with: 
1. identifying all of the offending font color contrast instances in the app
2. suggesting replacement colors and implementing them
3. testing to check for additions of new failing ront folor contrast

**Comment by jkleinsc** [comment]
@jglovier thanks for putting this together.  It is important work!

**Comment by Anisha1234** [comment]
@jglovier I would also like to work on the same issue

**Comment by jkleinsc** [comment]
@Anisha1234 please feel free to work on this issue.  I'll mark it as in progress.

**Comment by Anisha1234** [comment]
Thank you so much @jkleinsc . I will start working on it and keep you updated

**Comment by Anisha1234** [comment]
@jkleinsc @jglovier Should I make different PRs for changes in button style of different sections of the webapp for example - inventory , patients etc ?

**Comment by emadehsan** [comment]
By the way @jglovier @Anisha1234 have you considered usability for **Color Blind People** while choosing color schemes?

**Comment by Anisha1234** [comment]
@emadehsan what exactly do you want to be added for the same issue that you mentioned?

**Comment by emadehsan** [comment]
As you guys are working hard on UI and UX 👍, I was just asking that _Can a Colorblind person using the software distinguish between all the colors used throughout HospitalRun?_

**Comment by Anisha1234** [comment]
@emadehsan @jglovier @jkleinsc In my current PR #983 , I have not considered the case of colour blind people.But I would surely like to work on it to make the colour scheme more user friendly .
